### PR TITLE
Increase Feedback Input Limit

### PR DESCRIPTION
**Pull request by ABausG**
Like brought up in #75 the current limit might be to short for some Feedback and does not reflect what the backend is capable of. 
https://github.com/wiredashio/wiredash-sdk/issues/75#issuecomment-749762394

This Pullrequest increases the Limit to 2048 and shows a counter with the still allowed characters when getting close to it (currently starting with 50 left)

### CODE REVIEW

**Review by atoka93 — CHANGES_REQUESTED** [review]
Hey Anton, thank you for your pull request!

I have some suggestion to how to display the limit:

File: lib/src/feedback/components/input_component.dart [hunk 1]
```diff
@@ -150,12 +156,25 @@ class _InputComponentState extends State<InputComponent> {
     return null;
   }
 
+  Widget _getCounterText(BuildContext context,
+      {int currentLength, int maxLength, bool isFocused}) {
+    switch (widget.type) {
+      case InputComponentType.feedback:
+        final difference = maxLength - currentLength;
+        return difference <= _lengthWarningThreshold
+            ? Text(difference.toString())
```
[the remark this answers is not in the document]
Comment (atoka93) [reply lib/src/feedback/components/input_component.dart:165]: I'll do the changes so we can merge this ASAP.

**Comment by passsy** [comment]
It now shows:

`2040/2048` in grey
`2049/2048` in red


I adjusted the counter to count `characters` including emojis. `"👍👍👍".length` is 3, not 6

**Comment by passsy** [comment]
```
WiredashApiException{500, message: child "message" fails because ["message" length must be less than or equal to 2048 characters long], body: {"message":"child \"message\" fails because [\"message\" length must be less than or equal to 2048 characters long]"}
```

When I try to submit a 2048 character string including emojis

**Comment by passsy** [comment]
Character counting fix will land in backend. Doesn't matter for non-UTF-8 characters

**Review by passsy — APPROVED** [review]

**Review by atoka93 — APPROVED** [review]

**Pull request merged**